pkgpanda/cli.py

```diff
@@ -166,6 +168,17 @@ def main():
                 print(pkg)
             sys.exit(0)
 
+        if arguments['show']:
+            package_id = arguments['<package-id>']
+            try:
+                pkginfo = repository.load(package_id).pkginfo
+                print(json.dumps(pkginfo, indent=4))
+                sys.exit(0)
+            except PackageNotFound:
+                print("Package not found: {}".format(package_id),
+                      file=sys.stderr)
```

Comment:
To be consistent with the style in the rest of this file, the newline that breaks this up should go after `print(`. But it looks like this file has a line length limit of 120 columns (there are a few existing lines that go beyond 80), so you can go ahead and put this on one line.

Reply:
I think the idea behind having mandatory `pep8` linting on every PR is to be able to avoid these kind of style discussions, so given that this is valid according to https://www.python.org/dev/peps/pep-0008/#id17 I'm just going to leave it as is.

Reply:
From the doc you linked to:

> A style guide is about consistency. Consistency with this style guide is important. Consistency within a project is more important. *Consistency within one module or function is the most important.*

Emphasis mine. If you insist that pep8 catch all style inconsistencies before you make a change, then feel free to add the corresponding pep8 config to this PR. Otherwise, please make the simple style change I requested in order to keep this module consistent. Or find someone else to review this PR.

Reply:
To be clear, the portion you're quoting is defining exceptions for when you should *not* follow pep8, i.e. local consistency trumps the rules in this guide. Since we are enforcing pep8 in DC/OS, it is not really relevant here since all code is consistently pep8 anyways. Also, there's not a single instance of a `print(` followed by a newline in the current DC/OS codebase.

However, since you seem to care strongly about this, I went ahead and applied the style fix.